large_image/widgets/HistogramEditor.vue

```diff
@@ -1,33 +1,33 @@
 <script>
 const dtypeAliases = {
-    'i1': 'int8',
-    'u1': 'uint8',
-    'i2': 'int16',
-    'u2': 'uint16',
-    'i4': 'int32',
-    'u4': 'uint32',
-    'i8': 'int64',
-    'u8': 'uint64',
-    'f2': 'float16',
-    'f4': 'float32',
-    'f': 'float32',
-    'f8': 'float64',
-    'd': 'float64',
-}
+    i1: 'int8',
+    u1: 'uint8',
+    i2: 'int16',
+    u2: 'uint16',
+    i4: 'int32',
+    u4: 'uint32',
+    i8: 'int64',
+    u8: 'uint64',
+    f2: 'float16',
+    f4: 'float32',
+    f: 'float32',
+    f8: 'float64',
+    d: 'float64'
+};
 
 const dtypeRanges = {
-    'int8': [-128, 127],
-    'uint8': [0, 255],
-    'int16': [-32768, 32767],
-    'uint16': [0, 65535],
-    'int32': [-2147483648, 2147483647],
-    'uint32': [0, 4294967295],
-    'int64': [-9223372036854775808, 9223372036854775807],
-    'uint64': [0, 18446744073709551615],
-    'float16': [-65,504, 65,504],
-    'float32': [-3.4e+38, 3.4e+38],
-    'float64': [-1.7e+308, +1.7e+308],
-}
+    int8: [-128, 127],
+    uint8: [0, 255],
+    int16: [-32768, 32767],
+    uint16: [0, 65535],
+    int32: [-2147483648, 2147483647],
+    uint32: [0, 4294967295],
+    int64: [-9e18, 9e18],
+    uint64: [0, 1e19],
+    float16: [-65504, 65504],
```

Comment:
I'm curious where 65504 came from, since when I ask numpy I get 
```
>>> np.finfo(np.dtype('float16')).min
np.float16(-65500.0)
>>> np.finfo(np.dtype('float16')).max
np.float16(65500.0)
```

Reply:
My fault for taking AI overview at face value: 
![image](https://github.com/user-attachments/assets/18f18fe3-ac77-4eb6-82fb-beddd39e325e)
Let's go with what's reported by numpy. Changed in [4308f2e](https://github.com/girder/large_image/pull/1839/commits/4308f2ec93e55816d4b2543401149caf37e93712)

Reply:
And, we might want 2 significant figures for int64 and uint64 (or round to a less zero value), since with uint64 with lost nearly half the range.